presto-benchto-benchmarks/src/main/resources/sql/presto/tpch/q1.sql

```diff
@@ -10,7 +10,7 @@ SELECT
   avg(l.discount)                                       AS avg_disc,
   count(*)                                              AS count_order
 FROM
-  "${database}"."${schema}"."lineitem" AS l
+  "${database}"."${schema}"."${prefix}lineitem" AS l
```

Comment:
Later, when it comes to tpcds. At facebook, we can't easily create schemas ("namespaces"), they are managed and we have to put different scale factor tables in the same namespace.